addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -21,7 +21,6 @@ import (
 	"crypto/sha1"
 	"encoding/hex"
 	"fmt"
-
```

Comment:
follow import order https://github.com/golang/go/wiki/CodeReviewComments#imports

Reply:
somehow goland always messes this up. 
I just installed goimports. This should be fixed now